### ISSUE TITLE
Money.parse error with single-digit currency amounts

### ISSUE THREAD

**Issue by njwest** [kipcole9/money#107]
Not sure if this is by design, but when using Money.parse to create #Money from single-digit amounts, I get a Money.Invalid error. E.g. when I try to Money.parse USD1, the amount is invalid:

```elixir
iex(1)> Money.parse("USD1")
{:error, {Money.Invalid, "Could not parse \"USD1\"."}}

iex(2)> Money.parse("USD10")
#Money<:USD, 10>

iex(3)> Money.parse("USD1.00")
#Money<:USD, 1.00>

iex(4)> Money.parse("USD9")   
{:error, {Money.Invalid, "Could not parse \"USD9\"."}}

iex(5)> Money.parse("USD90")
#Money<:USD, 90>

iex(6)> Money.parse("USD9.0") 
#Money<:USD, 9.0>

iex(7)> Money.parse("USD9.00")
#Money<:USD, 9.00>

```

**Comment by kipcole9** [comment]
Pushed a release [version 4.2.1 to github](https://github.com/kipcole9/money/releases/tag/v4.2.1) and published [version 4.2.1 to hex](https://hex.pm/packages/ex_money/4.2.1).

Thanks for the report and sorry for the inconvenience.

**Comment by njwest** [comment]
Thanks a ton for the fix! No need to apologize... this package is incredibly convenient :P
